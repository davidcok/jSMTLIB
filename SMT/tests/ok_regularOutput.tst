; test changing the regular output channel
(set-logic QF_UF)
(get-option :regular-output-channel)
(set-option :regular-output-channel "tempout")
(get-option :regular-output-channel)
(assert true)
(set-option :regular-output-channel "stdout")
(get-option :regular-output-channel)
(assert true)
