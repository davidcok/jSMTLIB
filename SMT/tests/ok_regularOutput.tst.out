success
"stdout"
success
"stdout"
success
